trunk/src/modules/zoltan_wrapper.F90

```diff
@@ -117,8 +115,8 @@ subroutine zoltan_w_init()
       call zoltan_w_handle_err(ierr,'Zoltan_Set_Fn')
 !     Return lists of neighbors, their owners, and optionally face weights
 !     for elements sharing a face with a given element
-!      ierr = Zoltan_Set_Fn(zz,ZOLTAN_EDGE_LIST_FN_TYPE,zoltan_w_query_neig_list)
-      ierr = Zoltan_Set_Fn(zz,ZOLTAN_EDGE_LIST_MULTI_FN_TYPE,zoltan_w_query_neig_list_omp)
+      ierr = Zoltan_Set_Fn(zz,ZOLTAN_EDGE_LIST_FN_TYPE,zoltan_w_query_neig_list)
+!      ierr = Zoltan_Set_Fn(zz,ZOLTAN_EDGE_LIST_MULTI_FN_TYPE,zoltan_w_query_neig_list_omp)
```

Comment:
Is this change due to incorrectness of the omp routine?

Reply:
I think think I did have problems with it at some point; I can't say for sure though, I have a separate version in the MG solver so I haven't used it in a long time. Probably okay to revert this.

Reply:
Ok, I'll revert it for now, because I think the performance of the sequential version was quite bad.